actions/ctf-setup-run-tests-environment/action.yml

```diff
@@ -61,19 +61,29 @@ inputs:
   gati_token:
     required: false
     description: Token provided by GATI to pull from private repos
+  enable-gap:
+    required: false
+    default: "true"
+    description: Configure and enable GAP for accessing internal services.
   main-dns-zone:
-    required: true
+    required: false
+    default: ""
     description:
       The primary DNS zone used in the hostname of the Kubernetes API endpoint.
-  k8s-cluster-name:
-    required: true
-    description:
-      The name of the Kubernetes cluster to be used in the `kubectl`
-      configuration for accessing the desired cluster.
+      Required if GAP is enabled.
 
 runs:
   using: composite
   steps:
+    - name: Validate GAP inputs
+      shell: bash
+      if: inputs.enable-gap == 'true'
+      run: |
+        if [ -z "${{ inputs.main-dns-zone }}" ]; then
+          echo "Error: main-dns-zone is required when enable-gap is true."
```

Comment:
shouldn't we validate `k8s-cluster-name` as well?

Reply:
Thanks.